src/test/regress/wrap_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+cmd="$@"
+
+$cmd &
+regress_pid=$!
+
+# the regression tests don't take 60 seconds to run, we can time out if they do
+sleep 60 &
+sleep_pid=$!
+
+echo $regress_pid
+wait -n
+process_status=$?
+
+if $(kill -0 $regress_pid); then
+  echo "it's been 60 seconds and regress is still running"
+
+  printf 'is mitmdump running?\n'
+  ps aux | grep mitm
+
+  printf '\nthe proxy output:\n'
+  cat proxy.output
+
+  # what is mitmdump doing?
+  sudo timeout 1 strace -p `pgrep mitmdump`
```

Comment:
(a) not a fan of the `sudo`
(b) no `strace` on macOS
(c) no `pgrep` on macOS

Reply:
Hmmm, what do you suggest?

This is not intended to work on macOS, it's just there to debug the Travis failures. Hopefully it'll be gone by next week.

Do you know of a way to detect osx from inside a Makefile? I can only wrap if we're running on linux.

Reply:
Oh, so this is just for debugging? Maybe hide it behind an environment variable then? I dunno.

Reply:
Just pushed a commit which hides it behind an env variable, so the wrapper is only used on Travis and won't be hit when running things locally. That should make it safer to push to master.